tests/data/chrom_size/hg19.chrom.sizes

```diff
@@ -0,0 +1,93 @@
+chr1	249250621
+chr2	243199373
+chr3	198022430
+chr4	191154276
+chr5	180915260
+chr6	171115067
+chr7	159138663
```

Comment:
We just need chromosome 7 for the tests in this file.

Reply:
Ideally, we need to keep the tests down to a minimal size as possible.

Reply:
Only chr7 remains now.